lib/src/widget/input_history_text_field_state.dart

```diff
@@ -174,33 +180,31 @@ class InputHistoryTextFieldState extends State<InputHistoryTextField> {
   }
 
   Widget _badgeHistoryItem(item) {
-    return Flexible(
-      child: Container(
-        height: 32,
-        margin: EdgeInsets.only(right: 5, bottom: 5),
-        padding: EdgeInsets.only(top: 5, bottom: 5, left: 10, right: 10),
-        decoration: BoxDecoration(
-          color: this._backgroundColor(item) ??
-              // ignore: deprecated_member_use_from_same_package
-              widget.badgeColor ??
-              Theme.of(context).disabledColor.withAlpha(20),
-          borderRadius: BorderRadius.all(Radius.circular(90)),
-        ),
-        child: InkWell(
-          onTap: () => this._inputHistoryController.select(item.text),
-          child: Row(
-            mainAxisSize: MainAxisSize.min,
-            children: [
-              /// history icon
-              if (widget.showHistoryIcon) _historyIcon(),
+    return Container(
+      height: 32,
+      margin: EdgeInsets.only(right: 5, bottom: 5),
+      padding: EdgeInsets.only(top: 5, bottom: 5, left: 10, right: 10),
+      decoration: BoxDecoration(
+        color: this._backgroundColor(item) ??
+            // ignore: deprecated_member_use_from_same_package
+            widget.badgeColor ??
+            Theme.of(context).disabledColor.withAlpha(20),
+        borderRadius: BorderRadius.all(Radius.circular(90)),
+      ),
+      child: InkWell(
+        onTap: () => this._inputHistoryController.select(item.text),
+        child: Row(
+          mainAxisSize: MainAxisSize.min,
+          children: [
+            /// history icon
+            if (widget.showHistoryIcon) _historyIcon(),
 
-              /// text
-              _badgeHistoryItemText(item),
+            /// text
+            _historyItemText(item),
```

Comment:
I using _historyItemText directly because _badgeHistoryItemText only adds Flexible and  a redundant Container